### PR TITLE
Add UI services to the upgrade recipe

### PR DESCRIPTION
**Pull request by nacx**
/cc @chirauki

### CODE REVIEW

File: recipes/upgrade.rb [hunk 1]
```diff
@@ -27,16 +27,18 @@
...
-    services = %w{abiquo-aim}
-when "monitoring"
-    services = %w{abiquo-delorean abiquo-emmett}
-else
-    services = %w{abiquo-tomcat}
-end
+services = {
+    'monolithic'        => %w{abiquo-tomcat},
+    'remoteservices'    => %w{abiquo-tomcat},
+    'server'            => %w{abiquo-tomcat},
```
Comment (chirauki) [inline recipes/upgrade.rb]: Need to add httpd and websockify

File: recipes/upgrade.rb [hunk 2]
```diff
@@ -27,16 +27,18 @@
...
+    'kvm'               => %w{abiquo-aim},
+    'monitoring'        => %w{abiquo-delorean abiquo-emmett},
+    'ui'                => %w{apache2},
```
Comment (chirauki) [inline recipes/upgrade.rb]: It should be httpd?

**Comment by chirauki** [comment]
+1!

**Comment by nacx** [comment]
Kitchen and specs passing. Squashed and added notes to CHANGELOG before merging.

**Pull request merged**